src/expr/src/relation/func.rs

```diff
@@ -3244,13 +3261,11 @@ fn mz_acl_explode<'a>(
 pub enum TableFunc {
     AclExplode,
     MzAclExplode,
-    JsonbEach {
-        stringify: bool,
-    },
+    JsonbEach,
+    JsonbEachStringify,
     JsonbObjectKeys,
-    JsonbArrayElements {
-        stringify: bool,
-    },
+    JsonbArrayElements,
+    JsonbArrayElementsStringify,
```

Comment:
Please add `JsonbEachStringify` and `JsonbArrayElementsStringify` to `TableFunc::with_ordinality`.